components/brave_ads/browser/notification_helper_linux.cc

```diff
@@ -13,11 +12,19 @@ NotificationHelperLinux::NotificationHelperLinux() = default;
 
 NotificationHelperLinux::~NotificationHelperLinux() = default;
 
-bool NotificationHelperLinux::ShouldShowNotifications() const {
-  return !IsFullScreenMode();
+bool NotificationHelperLinux::ShouldShowNotifications() {
+  // TODO(tmancey): Investigate how we can detect if notifications are enabled
+  // within the Linux operating system
```

Comment:
Checking the policy with TODO's like this - would this be better as a new ticket?

Reply:
TODO's are allowed as long as a ticket has been raised, however good point so I have updated the TODO to contain a link to the issue